:q

```diff
@@ -0,0 +1,5 @@
+# This is Git's per-user configuration file.
+[user]
+# Please adapt and uncomment the following lines:
+	name = Kelli Brodhag
+	email = teacherkelli@gmail.com
```

Comment:
How do I delete this?

Reply:
Delete the file in Finder. It may appear as “/q” instead of “:q”. ([The reason for that oddity is complicated.](https://stackoverflow.com/questions/13298434/colon-appears-as-forward-slash-when-creating-file-name?answertab=votes#answer-13298479))